nutpie/sample.py

```diff
@@ -27,10 +27,12 @@ def sample(
     tune: int = 1000,
     chains: int = 4,
     seed: int = 42,
-    num_try_init=100,
+    num_try_init=200,
```

Comment:
Wow, that's high. I think we try like 10x times only in PyMC, but we do it per variable perhaps?

Reply:
I adapted this from stan a bit, they try even more often. I don't think it matters that much to be honest, but trying a few more times might make things a bit more robust, and it doesn't cost much...

Reply:
Yeah, I was not concerned, just surprised